src/libs/ReportUtils.ts

```diff
@@ -2904,6 +2912,26 @@ function buildOptimisticIOUReport(payeeAccountID: number, payerAccountID: number
     };
 }
 
+/**
+ * Populates the report field formula with the values from the report and policy.
+ * Currently, is supports only optimistic expense reports.
+ * Each formula field is either replaced with a value, or removed.
+ * If after all replacements the formula is empty, the original formula is returned.
```

Comment:
Let's also leave a link to the report fields here for anyone referencing this method 
https://help.expensify.com/articles/expensify-classic/insights-and-custom-reporting/Custom-Templates